doc/01_Installation_and_Upgrade/01_Upgrade_Notes.md

```diff
@@ -1,7 +1,9 @@
-# Upgrade Notes
+# Upgrade NotesAdded datetime resolve for metadata in assets
```

Comment:
I think this should be under 2.0.0 right?

Reply:
Oops, that shouldnt be there at all, must have been a copy&paste thing. Thanks!